llvm/test/Analysis/ScalarEvolution/pr120442.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: opt < %s -disable-output "-passes=print<scalar-evolution>" -scalar-evolution-max-iterations=0  -scalar-evolution-classify-expressions=0  2>&1 | FileCheck %s
+
+declare void @llvm.assume(i1)
+
+define void @pr120442() {
+; CHECK-LABEL: 'pr120442'
+; CHECK-NEXT:  Determining loop execution counts for: @pr120442
+; CHECK-NEXT:  Loop %bb2: backedge-taken count is i32 0
+; CHECK-NEXT:  Loop %bb2: constant max backedge-taken count is i32 0
+; CHECK-NEXT:  Loop %bb2: symbolic max backedge-taken count is i32 0
+; CHECK-NEXT:  Loop %bb2: Trip multiple is 1
+; CHECK-NEXT:  Loop %bb1: <multiple exits> Unpredictable backedge-taken count.
+; CHECK-NEXT:  Loop %bb1: Unpredictable constant max backedge-taken count.
+; CHECK-NEXT:  Loop %bb1: Unpredictable symbolic max backedge-taken count.
+bb:
+  call void @llvm.assume(i1 false)
+  call void @llvm.assume(i1 false)
+  br label %bb6
+
+bb1:
+  br label %bb2
+
+bb2:
```

Comment:
```suggestion
inner.header:
```

Reply:
Might be good to reorder and rename the blocks to make the sturcture easier to see in the test

Reply:
Thanks, simplified the test a bit more